content/ja/blog/2025/kubecon-japan.md

```diff
@@ -0,0 +1,52 @@
+---
+title: KubeCon + CloudNativeCon Japan 2025で、
+  OpenTelemetryのトークとアクティビティに参加しよう
+linkTitle: KubeCon Japan '25
+date: 2025-05-29
+author: '[Tiffany Hrabusa](https://github.com/tiffany76) (Grafana Labs)'
+default_lang_commit: a770dd9ad30d08571d02184b339441061f35e464
+# prettier-ignore
+cSpell:ignore: Baazi Chauhan Hrabusa Kang Kasper Mostafa Nissen Radwan Shivay Siddharth Vijay
+---
+
+OpenTelemetryプロジェクトは、[KubeCon + CloudNativeCon Japan][]（[参加登録][registration]）
+および東京で同時開催される[Community Day][]（2025年6月15日〜17日）で、OpenTelemetryコミュニティのメンバーと一緒に過ごすことを皆さんにご案内します。
+
+この記事では、KubeCon期間中に予定されているOpenTelemetry関連のすべてのアクティビティを紹介しています。カンファレンス開始前に、随時更新情報をチェックしてください！
+
+## Community Day
+
+これは6月15日に開催される **無料** のイベントで、本カンファレンスに先立ち、クラウドネイティブコミュニティと交流する貴重な機会です。
+
+[Community Day][]では、KubeCon + CloudNativeCon Japanの参加者と、さまざまな地域ミートアップコミュニティ、Special Interest Group（SIG）やCloud Native Community Japanのサブグループのメンバーが一堂に会し、クラウドネイティブエコシステムにおける協業、学び、貢献を促進します。
+このイベントは、クラウドネイティブ開発やコミュニティ参加のさまざまな側面を対象とする複数のテーマ・トラックに分かれて構成されています。
+今すぐ登録しましょう！
+
+## KubeConトーク一覧
```

Comment:
Add heading IDs when the title isn't all in English.

Reply:
@chalin added, thanks!